indico/modules/designer/templates/template.html

```diff
@@ -305,7 +304,7 @@
                 <div class="flexrow f-a-baseline">
                     <label for="element-list">{% trans %}Insert element:{% endtrans %}</label>
                     <select id="element-list">
-                        {% for group_id, group in placeholders.items() | sort(attribute='1.title') %}
+                        {% for group_id, group in placeholders.viewitems() | sort(attribute='1.title') %}
```

Comment:
That's Python-2-only...

Reply:
Yes I was in the process of reverting those lines

Reply:
What about two separate dictionaries? is it ok?